src/lwc/geTemplateBuilderFormField/geTemplateBuilderFormField.js

```diff
@@ -129,6 +129,38 @@ export default class geTemplateBuilderFormField extends LightningElement {
         return GeLabelService.format(this.CUSTOM_LABELS.geAssistiveFieldDown, [this.field.label]);
     }
 
+    /*******************************************************************************
+    * Start getters for data-qa-locator attributes
+    */
+
+    get qaLocatorInputFieldLabel() {
```

Comment:
Is it overkill to have these locators tracked as custom labels with separate getters? I'm wondering if we should be delivering this with just hard-coded attributes, or if there's a particular benefit to using labels